source/reviews/speed.txt

```diff
@@ -0,0 +1,149 @@
+================
+Speed of Reviews
+================
+
+Why Reviews Should Be Fast
+--------------------------
+
+**At MongoDB, we optimize for the speed at which a team of authors can produce
+content together**, as opposed to optimizing for the speed at which an
+individual author can write content. The speed of an individual is
+important, it's just not *as* important as the velocity of the entire team.
+
+When code reviews are slow, several things happen:
+
+-   **The velocity of the team as a whole is decreased.** Yes, the individual
+    who doesn't respond quickly to the review gets other work done. However, new
+    features and bug fixes for the rest of the team are delayed by days, weeks,
+    or months as each PR waits for review and re-review.
+-   **Authors start to protest the review process.** If a reviewer only
+    responds every few days, but requests major changes to the PR each time,
+    that can be frustrating and difficult for authors. Often, this is
+    expressed as complaints about how "strict" or "nitpicky" the reviewer
+    is being. If the reviewer requests the *same* substantial changes
+    (changes which really do improve content health), but responds *quickly*
+    every time the developer makes an update, the complaints tend to disappear.
+    **Most complaints about the code review process are resolved by making the
+    process faster.**
+-   **Content health can be impacted.** When reviews are slow, there is increased
+    pressure to allow developers to submit PRs that are not as good as they
+    could be. Slow reviews also discourage content cleanups, refactorings, and
+    further improvements to existing PRs.
+
+How Fast Should Code Reviews Be?
+--------------------------------
```

Comment:
Suggestion: Could a discussion be added about focus time? Personally, I've found that context switching between reviews and ticketed work makes my velocity on both slow.

Is batching reviews at EOD acceptable?